src/Input/FormField.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { bemClassNames } from '../utils';
+import './formfield.css';
+
+const bem = bemClassNames('d2ui-form-field');
+
+const FormField = ({
+    children,
+    valid,
+    warning,
+    disabled,
+    error,
+    dense,
+    fullWidth,
```

Comment:
Does `fullWidth` just mean we turn it into a `block`? Could we just call it `block`? :)